settings.py

```diff
@@ -2077,3 +2078,12 @@ def _pkce_required(client_id):
 # Disable Datadog trace startup logs by default
 # https://docs.datadoghq.com/tracing/troubleshooting/tracer_startup_logs/
 os.environ['DD_TRACE_STARTUP_LOGS'] = os.environ.get('DD_TRACE_STARTUP_LOGS', 'False')
+
+# Config settings for the google oauth handshake to get a user token
+# Google Cloud Platform secret settings config file
+GOOGLE_OATH_CONFIG = {}
```

Comment:
@RynhardPietersen can you include a reference to the commcare-cloud PR that will add the localsettings for this in the PR description? Thanks!